run_scmake.slurm

```diff
@@ -1,18 +1,18 @@
 #!/bin/bash
 #BATCH --job-name=national_flow_pull    # name that you chose
-#SBATCH -p UV,normal,short #the partition you want to use, for this case prod is best
+#SBATCH -p cpu
 #SBATCH -A iidd                 # your account
 #SBATCH -o shellLog/slurm-%A_%a.out
-#SBATCH --time=2:00:00        # time at which the process will be cancelled if unfinished
+#SBATCH --time=48:00:00        # time at which the process will be cancelled if unfinished
 #SBATCH --mail-type=ALL
 #SBATCH --export=ALL
 ##SBATCH --array=1        # process IDs 
 #SBATCH -n 1
 #SBATCH --mem=60GB
 
-module load singularity/3.4.1
+module load singularity
 
-srun singularity exec national-data-pulls_v0.0.sif Rscript -e '
+srun singularity exec national-data-pulls_v0.1.sif Rscript -e '
 library(scipiper);
 options(scipiper.getters_file = "10_nwis_pull.yml"); 
 scmake("20_data_munge/out/daily_flow.rds.ind")'
```

Comment:
@limnoliver and @padilla410 realizing now that an earlier decision when we set up this slurm script was to only build the `daily_flow` target. Thus, the `3_summarize` targets were not built nor were the `uv` data pulled. So while performance did improve, it did not improve quite as drastically as originally thought. Still almost cuts the time in half, though! 22 hrs for pulling daily flow to just 12.5 hrs!

Reply:
I think we should do a separate PR that changes this slurm script to just do `scmake()` and builds the new `uv` data + updates the `3_summarize` targets.

Reply:
`UV` was repulled because `daily_flow.rds` includes both `dv` and the summarized `uv` data. So I think the only time accounting is truly for summarizing, which like does not take long?

Reply:
Yeah, realized that after I posted so I crossed some stuff out. Running now :)

Reply:
Completed! Took about the same amount of time as noted in #4 for previous builds